src/deriveLinkDefaults.ts

```diff
@@ -158,9 +158,9 @@ export function initialLoader(datasets: ILinkConfig['datasets']) {
     return Promise.all(resolvedDataSets).then((data) => {
       const cleaned = data.filter((d): d is DataSet => d != null);
       cleaned.forEach((d) => {
-        if (d.params && !Array.isArray(d.params) && d.params._endpoint && d.params.regions) {
+        if (d.params) {
           // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
-          d.title = `${d.params._endpoint} | ${d.params.regions} | ${d.title}`;
+          d.title = `${Object.values(d.params).join(' > ')} > ${d.title}`;
```

Comment:
This is a definite improvement and works well for now, but i wouldnt call this a full solution for #42 ...  It lacks clarity and doesnt reproduce the normal/typical `LeftMenu` for a COVIDcast-type signal.

instead of:
`covidcast > doctor-visits > smoothed_cli > day > state > ar > value`,
it would be preferable to have something more like:
`covidcast > doctor-visits:smoothed_cli > state:ar`
for brevity and expressiveness

(lets leave it the way it is for now in this PR and return to it later when other higher-priority PRs are out of the way)

Reply:
I implemented this now, as it's fairly straightforward to modify the template. lmk what you think!

Reply:
Looks great for now. We should keep #42 open to later support reproducing the full form dropdown with value/stderr/sample_size.